frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick-start-devperspective.feature

```diff
@@ -11,80 +11,71 @@ Feature: Quick Starts card in developer console
+              And user clicks on the "View all quick starts" on Build with guided documentation card
+             Then user can see "Get started with a sample application" card
               And user can see time taken to complete the tour on the card
               And user can see Complete label
-              And user can see Review the tour link
 
 
         @to-do
-        Scenario: Quick tour page when tour is not completed: QS-03-TC04
-            Given user is in developer perspective
-              And user has not completed "Get started with a sample application" Quick Start
-             When user goes to +Add page
-              And user clicks on the "View all Quick Starts" on Quick Starts card
+        Scenario: Quick Starts page when Quick Start is not completed: QS-03-TC04
+            Given user has not completed "Get started with a sample application" Quick Start
+             When user goes to Add page
+              And user clicks on the "View all quick starts" on Build with guided documentation card
              Then user can see "Get started with a sample application" Quick Start card
-              And user can see time taken to complete the tour on the card
+              And user can see time taken to complete the quick start on the card
               And user can see In Progress label
-              And user can see Resume the tour and Restart the tour link
 
 
         @to-do
-        Scenario: Remove quick tour card from +Add view: QS-03-TC05
-            Given user is in developer perspective
-             When user goes to +Add page
-              And user clicks on the kebab menu in the Quick Start card
-              And user clicks on Remove Quick Starts
-             Then Quick Starts card will be removed from +Add page
+        Scenario: Hide Build with guided documentation card from Add view: QS-03-TC05
+            Given user is at Add page
+             When user clicks on the kebab menu in the Getting started resources card
+              And user clicks on Hide from view
+             Then Build with guided documentation card will be removed from Add page
 
 
         @to-do
-        Scenario: Quick Starts card links with status as in progress: QS-03-TC06
-            Given user is at +Add page
-             When user clicks on first Quick Starts link on the Quick Starts card
-              And user clicks on the Start tour
+        Scenario: Build with guided documentation card links with status as in progress: QS-03-TC06
+            Given user is at Add page
+             When user clicks on first Quick Starts link on the Build with guided documentation card
+              And user clicks on the Start button
               And user clicks on close button
-              And user clicks on leave the tour modal box
-             Then user can see "In Progress" status below the first Quick Starts link
+              And user clicks on Leave button in the Leave the tour modal box
```

Comment:
It would be better to format the text that your are using directly from the modals/buttons

```suggestion
              And user clicks on "Leave" button in the Leave the tour modal box
```

Reply:
We don't use "" every time for clicking on a button